### PR TITLE
refactor: re-use version no from package.json to maintain consistency

### PR DESCRIPTION
**Pull request by muke1908**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -13,14 +13,20 @@
   <div class="wrapper">
     <div id="root"></div>
   </div>
-  <script src="/web-sdk-0.0.2.min.js"></script>
+
+
```
Comment (atatter) [inline public/index.html]: would be nice to have consistent spacing by 1 empty line

File: webpack.dev.config.js [hunk 2]
```diff
@@ -24,7 +25,7 @@ module.exports = {
   },
   devtool: 'eval',
   output: {
-    filename: '[name]-0.0.2.min.js',
+    filename: `[name]-${version}.min.js`,
```
Comment (atatter) [inline webpack.dev.config.js]: no need for versions in development

File: public/index.html [hunk 3]
```diff
@@ -13,14 +13,20 @@
...
+  <% for (var chunk in htmlWebpackPlugin.files.chunks) { %>
+  	<script type="text/javascript" src="<%= htmlWebpackPlugin.files.chunks[chunk].entry %>"></script>
```
Comment (atatter) [inline public/index.html]: no need for this when code is injected to head

File: webpack.dev.config.js [hunk 4]
```diff
@@ -52,7 +53,8 @@ module.exports = {
       API_ROOT: JSON.stringify(API_ROOT)
     }),
     new HTMLWebpackPlugin({
-      template: './public/index.html'
+      template: './public/index.html',
+      inject:false
```
Comment (atatter) [inline webpack.dev.config.js]: inject it to head, no need for extra code in index.html

**Comment by atatter** [comment]
Did you forget to remove version number from webpack dev config?
Version number in package.json and package-lock.json should be 0.2.0 i think, @vlasn can confirm that

**Comment by muke1908** [comment]
@atatter I thought it's good to have a version there as well - might help in cache bursting.
Last time the version was `0.0.2` so I made it `0.1.0` as it's a minor change.
@vlasn Can you confirm the version no?

**Comment by vlasn** [comment]
Pretty sure it was at 0.0.2 the last time I looked at it:
https://cdn.kyc-pass.com/web-sdk-0.0.2.min.js

**Comment by atatter** [comment]
Oh my bad, the version was 0.0.2 indeed (did not expect minor version bump for changes that don't affect the end result/build). Using version numbers in development - wouldn't help with cache busting even if there was a problem with it (version number stays the same during development).

File: webpack.dev.config.js [hunk 5]
```diff
@@ -24,7 +25,7 @@ module.exports = {
   },
   devtool: 'eval',
   output: {
-    filename: '[name]-0.0.2.min.js',
+    filename: `[name].min.js`,
```
Comment (atatter) [inline webpack.dev.config.js]: no need for backtick

**Pull request merged**